apis/v1beta1/validation/gateway.go

```diff
@@ -40,7 +40,8 @@ var (
 
 // ValidateGateway validates gw according to the Gateway API specification.
 // For additional details of the Gateway spec, refer to:
-//  https://gateway-api.sigs.k8s.io/v1beta1/references/spec/#gateway.networking.k8s.io/v1beta1.Gateway
+//
+//	https://gateway-api.sigs.k8s.io/v1beta1/references/spec/#gateway.networking.k8s.io/v1beta1.Gateway
```

Comment:
This indentation and extra line break seem like they shouldn't be necessary, is this required for the linter update?

Reply:
Good catch. This looks strange and when I deleted the extra line and tab indent the linter didn't mind.